### DISCUSSION TITLE
Can Streamdeck-linux work with immutable OS such as Fedora kinoite?

### DISCUSSION THREAD

**Discussion by fliptoback** [streamdeck-linux-gui/streamdeck-linux-gui#184] [Q&A]
Hi, I am currently using Linux Mint, but am looking to switch to Fedora Kinoite soon - am wondering if the Streamdeck will work fine on an immutable os? Is there a different installation method for these? 

Thanks a bunch.

**Comment by fliptoback** [comment]
Hi Bro, thanks for that. I read the content of that #56 thread but the
thread has nothing to do with immutable linux distro?

**Reply by Danie10** [reply]
@fliptoback no sorry, I'd posted to wrong thread, but had immediately deleted it.

**Comment by coolapso** [comment]
not sure about fedora, but there's some documentation about nixOs: https://streamdeck-linux-gui.github.io/streamdeck-linux-gui/installation/nixos/

but the application should just work as any linux os as long as the necessary dependencies and its versions are available.

**Reply by fliptoback** [reply]
Thanks bro. I hope so too.

**Reply by coolapso** [reply]
> Thanks ~~bro~~. I hope so too.
